Headless virtual list (#702)
solidjs-community/solid-primitives: packages/virtual/README.md

[![version](https://img.shields.io/npm/v/@solid-primitives/virtual?style=for-the-badge)](https://www.npmjs.com/package/@solid-primitives/virtual)
[![stage](https://img.shields.io/endpoint?style=for-the-badge&url=https%3A%2F%2Fraw.githubusercontent.com%2Fsolidjs-community%2Fsolid-primitives%2Fmain%2Fassets%2Fbadges%2Fstage-0.json)](https://github.com/solidjs-community/solid-primitives#contribution-process)

> A headless `createVirtualList` utility function for [virtualized lists](https://www.patterns.dev/vanilla/virtual-lists/) and a basic, unstyled `VirtualList` component (which uses the utility).

Reviewer: You could make `createVirtualList` and `VirtualList` into section headers and add hash links here like in other packages.

Reviewer: this is what I meant:
```suggestion
- [`createVirtualList`](#createvirtuallist) - A headless  utility function for [virtualized lists](https://www.patterns.dev/vanilla/virtual-lists/)
- [`VirtualList`](#virtuallist) - a basic, unstyled component based on `createVirtualList`
```

## How to use it

> `createVirtualList` is a headless utility for constructing your own virtualized list components with maximum flexibility.

Reviewer: `createVirtualList` should be added to primitives list in `package.json`
